File: tests/go/cases/case_297_alias_hop_chain_09.go
package escape_tests

var retainedCase297 = []map[string]string{}

func Case297AliasHopChain09(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "alias_hop_chain_09",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "alias_hop_chain_09:" + raw,
	}
	a := payload
	b := a
	c := b
	// ESCAPE: multi-hop alias chain retained globally.
	retainedCase297 = append(retainedCase297, c)
	return "ok"
}

File: tests/nodejs/cases/case_297_alias_hop_chain_09.js
/**
 * Task case 297: alias_hop_chain_09 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case297AliasHopChain09(input) {
  const taskName = 'alias_hop_chain_09';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const a = payload;
  const b = a;
  const c = b;
  // ESCAPE: multi-hop alias chain into retained sink.
  retainedAudit.push(c);
  return 'ok';
}

module.exports = { case297AliasHopChain09 };
